PWGHF/Core/HfHelper.h

```diff
@@ -1336,6 +1336,65 @@ class HfHelper
     mlScores.push_back(candB.prong0MlScoreNonprompt()); // we want non-prompt for beauty
     return applySelectionDmesMlScoresForB(cuts, binsPtC, RecoDecay::pt(candB.pxProng0(), candB.pyProng0()), mlScores);
   }
+
+  /// Get candidate mass (ALICE3 HF data model)
+  /// \tparam TCand candidate type
+  /// \param cand candidate
+  /// \return candidate mass
+  template <o2::analysis::CharmHadAlice3 CharmHad, typename TCand>
+  static double getCandMass(const TCand& cand)
+  {
+    switch (CharmHad) {
+      case o2::analysis::CharmHadAlice3::Lc:
+        return cand.isSwapped() ? invMassLcToPiKP(cand)
+                                : invMassLcToPKPi(cand);
+      default:
+        LOG(fatal) << "Unsupported charm hadron type";
+        return -1.;
+    }
+  }
```

Comment:
These cannot be here. The best place would be to make dynamic columns `invMassLcTo...` in the ALICE3 HF data model or to make a separate header for ALICE3 HF utils and call `HfHelper::invMassLcTo...` from there.